proposals/webid-oidc/README.md

```diff
@@ -0,0 +1,267 @@
+# WebID-OIDC spec
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Motivation](#motivation)
+  - [Benefits and Capabilities](#benefits-and-capabilities)
+  - [If You're Unfamiliar with OIDC](#if-youre-unfamiliar-with-oidc)
+* [Differences from Classic OpenID Connect](#differences-from-classic-openid-connect)
+* [Brief Workflow Summary](#brief-workflow-summary)
+* [Enhancement: Add `webid` Claim to ID Token](#enhancement-add-webid-claim-to-id-token)
+* [Enhancement: WebID Provider Confirmation](#enhancement-webid-provider-confirmation)
+* [Detailed Sign In Workflow Example](#detailed-sign-in-workflow-example)
+
+## Introduction
+
+WebID-OIDC is an authentication delegation protocol (as well as a toolkit of
+useful auth-related verification techniques) suitable for WebID-based
+decentralized systems such as [Solid](https://github.com/solid/solid), as well
+as most LDP-based systems.
+
+The end result of any WebID authentication workflow is a verified WebID
```

Comment:
I think the way you said it is great. Verifying that an agent owns a WebID. Without the agent part "verified WebID URI" sounds like the use case for a certificate authority.